apiserver/facades/client/spaces/spaces.go

```diff
@@ -67,6 +73,16 @@ type Backing interface {
 
 	// AllMachines loads all machines
 	AllMachines() ([]Machine, error)
+
+	// RenameSpace renames the given space `a` given name `b`.
+	// SettingsChanges and ConstraintsBySpace holds the changes to be done in the state.
```

Comment:
update comment:
`// SettingsChanges and constraints holds the changes to be added to the state.`